po/it.po

```diff
@@ -2,21 +2,21 @@
 #
 # Italian translation for systemd package
 # Traduzione in italiano per il pacchetto systemd
-# Daniele Medri <dmedri@gmail.com>, 2013-2020.
+# Daniele Medri <dmedri@gmail.com>, 2013-2021.
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd\n"
-"Report-Msgid-Bugs-To: https://github.com/systemd/systemd/issues\n"
-"POT-Creation-Date: 2020-08-19 18:02+0200\n"
-"PO-Revision-Date: 2020-10-03 09:52+0200\n"
+"Report-Msgid-Bugs-To: \n"
```

Comment:
Why was the URL removed?

Reply:
Incidental poedit removal. I'll restore it. Thank you